internal/e2e/testdata/metric_stability_test_readers.yaml

```diff
@@ -0,0 +1,41 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: localhost:4318
```

Comment:
could the port be replaced with an env var that gets passed in? i worry that this test will be flaky due to port conflicts